index.js

```diff
@@ -638,8 +638,9 @@ var Client = module.exports = function(config) {
             var basic;
             switch (this.auth.type) {
                 case "oauth":
-                    path += (path.indexOf("?") === -1 ? "?" : "&") +
-                        "access_token=" + encodeURIComponent(this.auth.token);
+                    headers.authorization = " token " + this.auth.token;
```

Comment:
The value is a bit weird, normally you don't put space in the beginning, and token type being "token" instead of typical "Bearer" is also unusual.

Reply:
Not sure about the space, but "token" is what they want us to send https://github.com/peachworks/node-github/pull/3#pullrequestreview-731775896

Reply:
```suggestion
                    headers.authorization = "token " + this.auth.token;
```